File: java/src/main/java/codingdojo/Response.java
package codingdojo;

/**
 * Sent to the client to indicate whether the new product was added successfully.
 */
public class Response {
    private final int productId;
    private final int statusCode;
    private final String errorMessage;

    public Response(int productId, int statusCode, String errorMessage) {

        this.productId = productId;
        this.statusCode = statusCode;
        this.errorMessage = errorMessage;
    }

    public int getProductId() {
        return productId;
    }

    public int getStatusCode() {
        return statusCode;
    }

    public String getErrorMessage() {
        return errorMessage;
    }

    @Override
    public String toString() {
        return "Response{" +
                "productId=" + productId +
                ", statusCode=" + statusCode +
                ", errorMessage='" + errorMessage + '\'' +
                '}';
    }
}

File: csharp/ValidationProject/Response.cs
namespace Validation
{
    public class Response
    {
        public int ProductId { get; }
        public int StatusCode { get; }
        public string Message { get; }

        public Response(int productId, int statusCode, string message)
        {
            ProductId = productId;
            StatusCode = statusCode;
            Message = message;
        }

        public override string ToString()
        {
            return "Response{" +
                   "productId=" + ProductId +
                   ", statusCode=" + StatusCode +
                   ", message='" + Message + '\'' +
                   '}';
        }
    }
}